### PR TITLE
Update LastPass and CloudFlare WARP references

### PR DESCRIPTION
**Pull request by lannon**
* Replace link to deleted CloudFlare WARP Google Doc with getting started web page
* Remove outdated reference to LastPass availability via Self Service App

### CODE REVIEW

File: security/devices.md [hunk 1]
```diff
@@ -60,4 +60,4 @@ Additional guidelines apply to company laptops; you should:
 ## Network :satellite:
 In general, we operate with a stance of [Zero Trust](https://www.cloudflare.com/learning/security/glossary/what-is-zero-trust/).  We assume that our internal networks could be compromised and seek to secure resources rather than networks.  Given the nature of our [remote workforce]({{ site.baseurl }}{% link work/index.md %}) and our utilization of cloud services, we cannot assume that resources are located within an enterprise-owned network boundary.
 
-To help reduce networking risks, we utilize [Cloudflare for Teams](https://www.cloudflare.com/teams/).  Everyone should install and utilize the Warp client (instructions can be found [here](https://docs.google.com/document/d/1gd_BvZug6p0SD9j77njP3fSa33nyM3cNOMAbqJ5C7fA/edit)).
+To help reduce networking risks, we utilize [Cloudflare for Teams](https://www.cloudflare.com/teams/).  Everyone should install and utilize the Warp client (downloads can be found [here](https://developers.cloudflare.com/warp-client/get-started/)).
```
Comment (bmuller) [inline security/devices.md:63]: Most people won't be able to install directly due to local permissions (though they can use the self service app)

```suggestion
To help reduce networking risks, we utilize [Cloudflare for Teams](https://www.cloudflare.com/teams/).  Everyone should install and utilize the Warp client via the Self Service app.
```

File: security/processes.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ Do not sign up for a new service on behalf of Parallel without first receiving p
 This includes signing into web pages and web apps using your @parallelmarkets.com Google sign-in.
 
 ### Passwords
-Use a password manager!  A password manager let you memorize one strong password, and then it generates and manages strong, unique passwords for every site for which you have a login.  Employees can install [LastPass](https://www.lastpass.com/) via the Self Service App that is installed on your company device.
+Use a password manager!  A password manager let you memorize one strong password, and then it generates and manages strong, unique passwords for every site for which you have a login.  Employees can install [LastPass](https://www.lastpass.com/), which should be preloaded on company Chromebooks.
```
Comment (bmuller) [inline security/processes.md:26]: It should be there in Chrome on all company machines (including Macbooks)
```suggestion
Use a password manager!  A password manager let you memorize one strong password, and then it generates and manages strong, unique passwords for every site for which you have a login.  Employees can install [LastPass](https://www.lastpass.com/), which should be preloaded on company devices. 
```

File: security/devices.md (hunk 1 above)
Comment (lannon) [reply security/devices.md:63]: Good call. Didn't see it in Self Service initially, but I do now.

**Comment by bmuller** [comment]
Hey @lannon - I think this is ready to go whenever you're ready

**Comment by lannon** [comment]
@bmuller Agreed! I don't seem to have permission to merge it.

**Comment by bmuller** [comment]
Hmm - I'll check and see if that's intentional 😬

**Pull request merged**